.gitignore

```diff
@@ -5,4 +5,5 @@
 /*.log
 /*.pdf
 /*.toc
+/changes.texi
```

Comment:
Why is changes.texi ignored? There's also a discrepancy between lines added and lines removed, I think because of this.